website/content/api-docs/secret/databases/index.mdx

```diff
@@ -117,6 +122,17 @@ $ vault write database/config/mysql \
     password="secretpassword"
 ```
 
+### Sample ADO-style Request
```

Comment:
nit: Maybe "Sample CLI Request with ADO-style `connection_url`"? Or maybe we could add a blurb under this heading that adds a bit of detail on what this is demonstrating and why we set `disable_escaping`?

Reply:
update -- since we have included [this section](https://github.com/hashicorp/vault/pull/14260/files#diff-9f5ba0bda172ce5ac29bbdc128b6e19ae44f5b809264b852ede2671b28d2f349R196) that details the `disable_escaping` flag at the bottom of this file maybe we don't even need this section here?

Reply:
Personally I think the example should still be present here in the api docs, since the other file is in the general docs, but I'm open to more input.

Reply:
That makes sense to me. Maybe we could link to the general docs description of `disable_escaping` from here?

Reply:
Good idea!

Reply:
I added it above the snippet, but after looking at it I think it actually fit better to add within the parameter description. Let me know what you think: https://vault-4cepfxddf-hashicorp.vercel.app/api-docs/secret/databases#disable_escaping